C-Plus-Plus/cp/Rat in a maze.cpp

```diff
@@ -0,0 +1,118 @@
+/* C++ program to solve Rat in 
```

Comment:
Describe the Problem in detail. If possible add a Problem Link on some Online Coding Portal.